src/main/drivers/serial_uart.c

```diff
@@ -297,6 +297,67 @@ static void uartWrite(serialPort_t *instance, uint8_t ch)
     }
 }
 
+static void uartBeginWrite(serialPort_t *instance)
+{
+    uartPort_t *uartPort = (uartPort_t *)instance;
+
+    // Check if the TX line is being pulled low by an unpowered peripheral
+    if (uartPort->checkUsartTxOutput && !uartPort->checkUsartTxOutput(uartPort)) {
+        // TX line is being pulled low, so don't transmit
+        return;
+    }
```

Comment:
```suggestion
    // Check if the TX line is being pulled low by an unpowered peripheral
    if (uartPort->checkUsartTxOutput) 
        uartPort->checkUsartTxOutput(uartPort);
    }
```

check is called for side-effect only, data are buffered anyway

Reply:
Agreed. That was a lazy copy/paste.